### PR TITLE
Updated contact information and example JIDs

### PR DESCRIPTION
**Pull request by PeterWaher**
New contact information has been provided, and example JIDs are updated
to point to example.org.

### CODE REVIEW

File: xep-0332.xml [hunk 1]
```diff
@@ -30,11 +30,28 @@
         <author>
             <firstname>Peter</firstname>
             <surname>Waher</surname>
-            <email>peter.waher@clayster.com</email>
+            <email>peterwaher@hotmail.com</email>
             <jid>peter.waher@jabber.org</jid>
             <uri>http://www.linkedin.com/in/peterwaher</uri>
         </author>
 		<revision>
+			<version>0.5</version>
```
Comment (linuxwolf) [inline xep-0332.xml]: These two new revisions should be collapsed into a single revision (0.4).

**Comment by linuxwolf** [comment]
Given how many documents you have, I think it would be best if you used an entity reference for your author info.  See &stpeter; or &linuxwolf; in xep.ent (and various XEPs) for examples.

File: xep-0332.xml (hunk 1 above)
Comment (PeterWaher) [inline xep-0332.xml]: Removed the duplicate revision statement.

**Comment by linuxwolf** [comment]
Merged collapsed as b01bdca.

**Pull request closed without merging**